modules/currency-l1/src/main/scala/org/tessellation/currency/l1/cli/method.scala

```diff
@@ -60,7 +60,8 @@ object method {
         pullTxsCount = 100L
       ),
       healthCheck = healthCheckConfig(false),
-      collateral = collateralConfig(environment, collateralAmount)
+      collateral = collateralConfig(environment, collateralAmount),
+      doubleSignDetectProgram = DoubleSignDetectProgramConfig(1.minute)
```

Comment:
Please follow the naming we already have in the project

Reply:
`DoubleSignDetectProgramConfig` has been renamed `DoubleSignDetect`. `doubleSignDetectProgram` has been renamed `doubleSignDetect`.